src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -595,7 +594,7 @@ class FromUnpivotExpressionSegment(BaseSegment):
     )
 
 
-@snowflake_dialect.segment()
+@snowflake_dialect.segment(replace=True)
 class SamplingExpressionSegment(BaseSegment):
     """A sampling expression."""
 
```

Comment:
Just below this line is this:

```py
type = "snowflake_sample_expression"
```

Since we now support this in more dialects I think we should just change this to the more generic:


```py
type = "sample_expression"
```

Even if there are unique things to snowflake in this (we don't called it snowflake_from_expression even if slightly different).

Reply:
FIxed in [0c4b5579b12060f4de698d3ddc7946248f0216b3](https://github.com/sqlfluff/sqlfluff/pull/1811/commits/0c4b5579b12060f4de698d3ddc7946248f0216b3)